etc/adminhtml/system.xml

```diff
@@ -224,6 +224,42 @@
             <field id="enabled">1</field>
           </depends>
         </field>
+        <field
+                id="autocomplete_override"
+                type="textarea"
+                translate="label,comment"
+                sortOrder="80"
+                showInDefault="1"
+                showInWebsite="1"
+                showInStore="1"
+        >
+          <label>Autocomplete Override</label>
+          <comment>
+            Your User Token can be found on your accounts page. This is not required for your address validation to function. It is only required if you need to retrieve usage statistics on your API Key.
```

Comment:
"Advanced: Custom override code for Address Finder. Please see our documentation for how to use this"